subscriptions-matcher/src/main/java/io/arlas/subscriptions/service/SubscriptionsService.java

```diff
@@ -66,7 +65,7 @@
                     result.addAll(items.getHits());
                     next = items.getLinks() != null ? items.getLinks().get("next") : null;
                     if (next != null) {
-                        queryParams = getQueryParams(next.getHref().split("\\?")[1]);
+                        queryParams = getQueryParams(next.getHref().split("\\?")[1].replace("before=&", ""));
```

Comment:
Where does the `before` come from?

Reply:
It's a bug from ARLAS server. I replace it as a workaround until a patched version is released.